script.js

```diff
@@ -21,3 +21,142 @@ cross.addEventListener('click', hideMenu);
 for (let i = 0; i < links.length; i += 1) {
   links[i].addEventListener('click', hideMenu);
 }
+
+// Projects Data to access Dynamically
+
+const projects = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    popup_name: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: './images/About1',
+    technologies: ['Ruby Rails', 'Javascript', 'css', 'html'],
+    live_link: '',
+    live_source: '',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    popup_name: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: './images/About1',
+    technologies: ['Ruby Rails', 'Javascript', 'css', 'html'],
+    live_link: '',
+    live_source: '',
+  },
```

Comment:
- [ ] Please know that the `name` and `popup_name` should be the same because that's the project's name, also for some reason the technologies in the popup screen aren't the ones on the project card:
![image](https://user-images.githubusercontent.com/20567503/195331501-6c0fdf53-3b8e-415e-a18c-c2c542c9d555.png)
![image](https://user-images.githubusercontent.com/20567503/195331506-a61ce974-8563-43c7-9781-a16eee43296a.png)

You should put the same technologies from the objects array you're using, the same goes for the image it should be the same as well, the logic here is that the project cards and the popup both get the information from the objects array, kindly fix these issues and your project should be done! 😁

Reply:
My template doesn't have images in cards but I added images of portfolio images of the mobile version and desktop version for **Popup window**.

- Alternate image is added for each different card.
- Technologies error is also fixed, now technologies are as per cards. 

Please review the above changes, different images can be added to the update path for **Popup window**.